recipe/meta.yaml

```diff
@@ -47,7 +45,6 @@ extra:
   recipe-maintainers:
     - xylar
     - jreback
-    - leahecole
```

Comment:
I don't believe this maintainer ever accepted the invitation to join conda-forge.